src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -285,6 +360,83 @@ public Builder schema(@Nullable String schema) {
             return this;
         }
 
+        /**
+         * Configure ssl usage.
+         *
+         * @param ssl
+         * @return this {@link Builder}
+         */
+        public Builder ssl(boolean ssl) {
+            this.ssl = ssl;
+            return this;
+        }
+
+        /**
+         * Configure ssl cert.
+         *
+         * @param sslCert
+         * @return this {@link Builder}
+         */
+        public Builder sslCert(@Nullable String sslCert) {
+            this.sslCert = sslCert;
+            return this;
+        }
+
+        /**
+         * Configure ssl HostnameVerifier.
+         *
+         * @param sslHostnameVerifier
+         * @return this {@link Builder}
+         */
+        public Builder sslHostnameVerifier(@Nullable String sslHostnameVerifier) {
+            this.sslHostnameVerifier = sslHostnameVerifier;
+            return this;
+        }
+
+        /**
+         * Configure ssl key.
+         *
+         * @param sslKey
+         * @return this {@link Builder}
+         */
+        public Builder sslKey(@Nullable String sslKey) {
+            this.sslKey = sslKey;
+            return this;
+        }
+
+        /**
+         * Configure ssl mode.
+         *
+         * @param sslMode
+         * @return this {@link Builder}
+         */
+        public Builder sslMode(SSLMode sslMode) {
+            this.sslMode = sslMode;
+            return this;
+        }
+
+        /**
+         * Configure ssl password.
+         *
+         * @param sslPassword
+         * @return this {@link Builder}
+         */
+        public Builder sslPassword(@Nullable String sslPassword) {
+            this.sslPassword = sslPassword;
+            return this;
+        }
+
+        /**
+         * Configure ssl root cert.
+         *
+         * @param sslRootCert
+         * @return this {@link Builder}
+         */
+        public Builder sslRootCert(@Nullable String sslRootCert) {
```

Comment:
Nit: Let's document that this should be a path to the SSL root/CA certificate in PEM format.